tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,313 @@
+
+    :CaseLevel: Integration
+    """
+    # rex interface
+    iface = target_sat.execute('ip route | grep default | cut --delimiter=" " --field 5')
+    # fill in the global registration form
+    with target_sat.ui_session() as session:
+        cmd = session.host_new.get_register_command(
+            {
+                'general.organization': module_org.name,
+                'general.location': smart_proxy_location.name,
+                'general.operating_system': default_os.title,
+                'general.activation_keys': module_ak_with_cv.name,
+                'advanced.update_packages': True,
+                'advanced.rex_interface': iface,
+                'general.insecure': True,
+            }
+        )
+    expected_pairs = [
+        f'organization_id={module_org.id}',
+        f'activation_keys={module_ak_with_cv.name}',
+        f'location_id={smart_proxy_location.id}',
+        f'operatingsystem_id={default_os.id}',
+        f'{default_smart_proxy.name}',
+        'insecure',
+        'update_packages=true',
+    ]
+    for pair in expected_pairs:
+        assert pair in cmd
+    # rhel repo required for insights client installation,
+    # syncing it to the satellite would take too long
+    rhelver = rhel_contenthost.os_version.major
+    if rhelver > 7:
+        rhel_contenthost.create_custom_repos(**settings.repos[f'rhel{rhelver}_os'])
+    else:
+        rhel_contenthost.create_custom_repos(
+            **{f'rhel{rhelver}_os': settings.repos[f'rhel{rhelver}_os']}
+        )
+    # make sure there will be package availabe for update
+    package, repo_url = (FAKE_1_CUSTOM_PACKAGE, settings.repos.yum_1['url']) if rhelver == 6 else (
+        FAKE_7_CUSTOM_PACKAGE, settings.repos.yum_3['url'])
+
+    rhel_contenthost.create_custom_repos(fake_yum=repo_url)
+    rhel_contenthost.execute(f'yum install -y {package}')
+    # run curl
+    result = rhel_contenthost.execute(cmd)
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    assert rhel_contenthost.subscribed
+
+    # Verify server.hostname and server.port from subscription-manager config
+    assert target_sat.hostname == rhel_contenthost.subscription_config['server']['hostname']
+    assert CLIENT_PORT == rhel_contenthost.subscription_config['server']['port']
+
+    # Assert that a yum update was made this day ("Update" or "I, U" in history)
+    timezone_offset = rhel_contenthost.execute('date +"%:z"').stdout.strip()
+    tzinfo = datetime.strptime(timezone_offset, '%z').tzinfo
+    result = rhel_contenthost.execute('yum history | grep U')
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    assert datetime.now(tzinfo).strftime('%Y-%m-%d') in result.stdout
+    # Set "Connect to host using IP address"
+    target_sat.api.Parameter(
+        host=rhel_contenthost.hostname,
+        name='remote_execution_connect_by_ip',
+        parameter_type='boolean',
+        value='True',
+    ).create()
```

Comment:
I think we don't need to connnect the host with IP to run JT, hostname should work
```suggestion

```

Reply:
It was not needed.